test/strategies/elkilpstrategy.js

```diff
@@ -0,0 +1,308 @@
+
+            // We check if balances are correct
+            // Since we have only one depositor, strategy contract and staking contract should both be back at 0
+            expect((await elkIlpStrategyV5.balanceOf(owner.address))).eq(0)
+            expect((await stakingContract.balanceOf(elkIlpStrategyV5.address))).eq(0)
+            // The total deposit should be back to its previous value
+            expect(BigNumber.from(totalDepositsAfterWithdraw)).eq(BigNumber.from(totalDepositsBefore));
+            // We should get a slightly bigger amount of ELP tokens since we got the coverage as well
+            expect(await elkPairContract.balanceOf(owner.address)).gt(amount);
+        })
+    })
+
+    describe("Test coverage on withdraw - two depositor", async () => {
+
+        beforeEach(async () => {
+            // Setup approval for one account
+            await wavaxTokenContract.approve(elkRouterContract.address, BigNumber.from("1000000000000000000000000"))
+            await elkTokenContract.approve(elkRouterContract.address, BigNumber.from("1000000000000000000000000"))
+            await elkPairContract.approve(elkIlpStrategyV5.address, BigNumber.from("1000000000000000000000000"))
+
+            await elkRouterContract.swapExactTokensForTokens(BigNumber.from("10000000000000000"), BigNumber.from("5000000000000000"), [wavaxTokenContract.address, elkTokenContract.address], owner.address, 1807909162115)
+            await elkRouterContract.addLiquidity(wavaxTokenContract.address, elkTokenContract.address, await wavaxTokenContract.balanceOf(owner.address), await elkTokenContract.balanceOf(owner.address), 0, 0, owner.address, 1807909162115)
+
+            // Setup approval for the other account
+            await wavaxTokenContract.connect(account1).approve(elkRouterContract.address, BigNumber.from("1000000000000000000000000"))
+            await elkTokenContract.connect(account1).approve(elkRouterContract.address, BigNumber.from("1000000000000000000000000"))
+            await elkPairContract.connect(account1).approve(elkIlpStrategyV5.address, BigNumber.from("1000000000000000000000000"))
+            await elkRouterContract.connect(account1).swapExactTokensForTokens(BigNumber.from("10000000000000000"), BigNumber.from("5000000000000000"), [wavaxTokenContract.address, elkTokenContract.address], account1.address, 1807909162115)
+            await elkRouterContract.connect(account1).addLiquidity(wavaxTokenContract.address, elkTokenContract.address, await wavaxTokenContract.balanceOf(account1.address), await elkTokenContract.balanceOf(account1.address), 0, 0, account1.address, 1807909162115)
+        })
+
+        it('Withdraw correct elp_balance_account1 of ELK Liquidity token from the strategy', async () => {
+            // We get the elp_balance_account1 of ELK token we have
+            const elp_balance_account1 = (await elkPairContract.balanceOf(owner.address)).toString();
```

Comment:
sometimes you use snake case, sometimes you use camel case for variable names formating, I'd advise to keep it consistent on camel case as it's the most common one for js... I'm a victim of this problem too as I'm a main python dev

Reply:
Oops I forgot this one ^^ I've been coding in Rust lately and started to mix a bit everything ^^'

Reply:
Should be cool